modules/comment/comment.model.js

```diff
@@ -3,16 +3,13 @@ const mongoose = require('mongoose');
 const commentSchema = new mongoose.Schema({
   text: {
     type: String,
-    required: Boolean,
+    required: true,
   },
   date: {
     type: Date,
     default: Date.now,
   },
-  user: {
-    type: mongoose.Schema.Types.ObjectId,
-    ref: 'User',
-  },
+  email: String,
```

Comment:
Hey, I think, we have decided to have `user` object and nested there `{ email: String, name: String: adminId: ObjectId }`. Please, refactor it in the GraphQL model too

Reply:
I must have been absent when this was decided. Having said that, including name is a good idea, I'll do that. `adminId` on the other hand I don't quite understand - what is it supposed to be referring to and for what purpose?